pages/workshops.vue

```diff
@@ -235,9 +235,9 @@
               </figure>
               <div class="has-text-grey">
                 <span class="is-size-5">
-                  Sukhgij Ysothonsreekul, Ph.D.
+                  สุขกิจ ยโสธรศรีกุล, Ph.D.
```

Comment:
Check the spelling

Reply:
![Screen Shot 2564-04-08 at 13 49 01](https://user-images.githubusercontent.com/82072741/113980899-356d1600-9871-11eb-8642-50224d390d07.png)